shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -79,8 +67,8 @@
  * Manage FATE transactions
  */
 public class FateCommand extends Command {
-
-  private final static Logger LOG = LoggerFactory.getLogger(FateCommand.class);
+  private final String warning =
```

Comment:
make static?

Reply:
Fixed in [5be3d7c](https://github.com/apache/accumulo/pull/2914/commits/5be3d7caf039bccecd4539d8ffbda331d3b32132)